nipype/workflows/smri/freesurfer/autorecon1.py

```diff
@@ -26,9 +28,11 @@ def checkT1s(T1_files, cw256=False):
     origvol_names = ["{0:03d}.mgz".format(i + 1) for i in range(len(T1_files))]
 
     # check if cw256 is set to crop the images if size is larger than 256
-    if not cw256 and any(dim > 256 for dim in shape):
+    if not cw256 and (voxel_size[0] * t1_size[0] > 256) or (voxel_size[1] * t1_size[1] > 256) or (voxel_size[2] * t1_size[2] > 256):
         print("Setting MRI Convert to crop images to 256 FOV")
         cw256 = True
+    else:
+        print("No need to add -cw256 flag")
```

Comment:
Actually, For the logging bug, my colleague has posted this like 2 months ago, and you just closed this issue, here is the issue number [https://github.com/nipy/nipype/issues/1667](#1667)